code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm

```diff
@@ -240,3 +241,16 @@
 	new /obj/item/clothing/mask/gas/clownmaskpsyche(src.loc)
 	new /obj/item/clothing/shoes/clownshoespsyche(src.loc)
 	qdel(src)
+
+
+/mob/living/simple_animal/hostile/retaliate/cluwne/tempcluwne
+	//this version of a cluwne  is for when someone is temporarily turned into a cluwne but you don't intend for them to die before the transformation is finished
+	maxHealth = 500
+	health = 500
+	bookgib = 0
+
+/mob/living/simple_animal/hostile/retaliate/cluwne/tempcluwne/attackby(var/obj/item/O as obj, var/mob/user as mob)
+	var/currenthealth = health
+	playsound(src, 'sound/items/bikehorn.ogg', 50, 1)
+	health = currenthealth
```

Comment:
?

Reply:
yeah I don't know what that does but I saw it in line 99 in the original cluwne attackby proc so I just blindly copy pasted it over. The point of the tempcluwne is to mimic the cluwne in all ways except the instant bookgibbing in case someone abuses it to instantly kill transmogged players

Reply:
also by giving it plenty extra HP just in case

Reply:
1. The order of calls isn't correct to mimic the cluwne behavior. The supercall would have to be between these lines.
2. The supercall in here means the parent proc also gets called anyway, meaning you're playing the sound and setting the health twice, and if it weren't for the `bookgib` var, this proc wouldn't prevent books from gibbing this.

You already included the `bookgib` var. Just delete this whole proc.